app/scripts/filters/util.js

```diff
@@ -356,6 +356,11 @@ angular.module('openshiftConsole')
       return Navigate.resourceURL(resource, kind, namespace, null, {apiVersion: apiVersion});
     };
   })
+  .filter('navigateToTabURL', function (Navigate) {
+    return function(resource, tab) {
```

Comment:
Worth a comment here that `resource` must be the the API object itself since it can be either the object or name using the other `navigateResourceURL` filter